west.yml

```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 5acae16d9ea808785279b1de1cd177108c1b1354
+      revision: pull/476/head
```

Comment:
Good news, we've managed to get `sdk-zephyr` PR before the upmerge, please update SHA :
`revision: 37430f7c6b7b1e075fff26961e8dcfab12b6c800`

Reply:
done. thanks!